addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -363,9 +363,10 @@ export class PosStore extends Reactive {
 
         if (ids.size > 0) {
             await this.data.call("pos.order", "action_pos_order_cancel", [Array.from(ids)]);
+            return true;
         }
 
-        return true;
+        return false;
```

Comment:
Check if you can just return true here

Reply:
No it should be false. For example, if the `_onBeforeDeleteOrder` fail, the order will not go in the loop and thus not fill the ids list nor return false. It will thus not delete the order but still return true if we do not change it to false. So imo I should keep it as `return false`